### ISSUE TITLE
Website: add Contact / Friend list

### ISSUE THREAD

**Issue by agnag** [Neos-Metaverse/NeosPublic#3633]
### Is your feature request related to a problem? Please describe.

Add the friends / contact list to the website as well, so one can easily check who of my friends are online without launching Neos

### Describe the solution you'd like

a page when logged into the Neos Account on the website to see contacts, especially those online, and maybe where they are (world  / instance name, or private instance)

Similar to VRChat

Possibly also allow chatting from the website.

### Describe alternatives you've considered

-

**Comment by Goofybud16** [comment]
This seems related to #156

**Comment by LexiBasilisk** [comment]
Yeah- this issue is highly related/overlapped. As per #156 - it'd first require stability and documentation of the API- as per Frooxius' recommendation on that issue:
> We don't have plans to document the API at the moment, as that would require stabilizing it and supporting officially. There are some major planned changes to it first before opening it to public officially.
